### PR TITLE
Feature/application owner settings

### PR DESCRIPTION
**Pull request by SandGrainOne**
## Description
Adding support for application owner configuration (or preferences)

## Related Issue(s)
- #215 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Altinn.Notifications.Persistence/Repository/ApplicationOwnerConfigRepository.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    /// Initializes a new instance of the <see cref="ApplicationOwnerConfigRepository"/> class.
+    /// </summary>
+    /// <param name="dataSource">The npgsql data source.</param>
+    public ApplicationOwnerConfigRepository(NpgsqlDataSource dataSource)
+    {
+        _dataSource = dataSource;
+    }
+
+    /// <inheritdoc/>
+    public async Task<ApplicationOwnerConfig?> GetApplicationOwnerConfig(string orgId)
```
Comment (acn-sbuad) [inline src/Altinn.Notifications.Persistence/Repository/ApplicationOwnerConfigRepository.cs:39]: I would like to see some caching in front of this method as the data won't change that often we could rather invalidate the cache on insert. ( We will of course get some edge cases for cache across multiple pods, but that is something we can live with for now until we hopefully get a common cache across pods)

File: test/Altinn.Notifications.IntegrationTests/Notifications/EmailNotificationsOrderController/EmailNotificationOrdersControllerTests.cs [hunk 2]
```diff
@@ -171,6 +171,7 @@ public async Task Post_ServiceReturnsError_ServerError()
 
         // Act
         HttpResponseMessage response = await client.SendAsync(httpRequestMessage);
+        string respoonseString = await response.Content.ReadAsStringAsync();
```
Comment (acn-sbuad) [inline test/Altinn.Notifications.IntegrationTests/Notifications/EmailNotificationsOrderController/EmailNotificationOrdersControllerTests.cs:174]: Can this line be deleted? If not, typo should be fixed and string value used in some sort of assertion

**Comment by SandGrainOne** [comment]
Postponing work untill we've found a solution to the FromAddress issue.
